sources/Debate-frontend/src/app/loginpage/registration/registration.component.html

```diff
@@ -1,21 +1,23 @@
 <div class="registrationContent">
-    <form class="reg-form" #regform="ngForm" (ngSubmit)="onSubmit()">
-        <mat-form-field class="example-full-width">
-            <mat-label>Login</mat-label>
-            <input matInput placeholder="Podaj swój login" required [(ngModel)]="model.login" name="login">
-        </mat-form-field>
-        <mat-form-field class="example-full-width">
-            <mat-label>Hasło</mat-label>
-            <input matInput type="password" placeholder="Wprowadź swoje hasło" required [(ngModel)]="model.password" name="password">
-        </mat-form-field>
-        <mat-form-field class="example-full-width">
-            <mat-label>Powtórz hasło</mat-label>
-            <input matInput type="password" placeholder="Wprowadź swoje hasło ponownie" required [(ngModel)]="passwordCheck" name="passwordCheck">
-        </mat-form-field>
-        <mat-form-field class="example-full-width">
-            <mat-label>Email</mat-label>
-            <input matInput type="email" placeholder="Podaj swój email" required [(ngModel)]="model.email" name="email">
-        </mat-form-field>
-        <button mat-raised-button color="primary" [disabled]="!regform.form.valid" type="submit" style="margin-bottom: 7em; width: 100%; margin-left: 5px;" >Zarejestruj</button>
-    </form>
+    <div class="regCard">
+        <form class="reg-form" #regform="ngForm" (ngSubmit)="onSubmit()">
+            <mat-form-field class="example-full-width">
+                <mat-label>Login</mat-label>
+                <input matInput placeholder="Podaj swój login" required [(ngModel)]="model.login" name="login">
+            </mat-form-field>
+            <mat-form-field class="example-full-width">
+                <mat-label>Hasło</mat-label>
+                <input matInput type="password" placeholder="Wprowadź swoje hasło" required [(ngModel)]="model.password" name="password">
+            </mat-form-field>
+            <mat-form-field class="example-full-width">
+                <mat-label>Powtórz hasło</mat-label>
+                <input matInput type="password" placeholder="Wprowadź swoje hasło ponownie" required [(ngModel)]="passwordCheck" name="passwordCheck">
+            </mat-form-field>
+            <mat-form-field class="example-full-width">
+                <mat-label>Email</mat-label>
+                <input matInput type="email" placeholder="Podaj swój email" required [(ngModel)]="model.email" name="email">
```

Comment:
```suggestion
                <input matInput type="email" placeholder="Podaj swój email" required email [(ngModel)]="model.email" name="email">
```

Add Angular-provided email validator